profiles/base/package.use.mask

```diff
@@ -7,6 +7,10 @@
 # This file is only for generic masks. For arch-specific masks (i.e.
 # mask everywhere, unmask on arch/*) use arch/base.
 
+# Bernd Waibel <waebbl@gmail.com> (11 Nov 2018)
+# masked for testing
```

Comment:
why do you mask existing flags for everyone?

Reply:
so, the other part of this PR was merged now. I still find it odd to mask these flags after they have been available to the odd user in ~arch for a long time already. stable-mask them in case you planned to stabilise alembic, otherwise leave it alone?

Reply:
I hope I can stabilize this package at some time, so I moved the masking to package.use.stable.mask, hoping this being the correct location. If it's not, feel free to ping me again.